### DISCUSSION TITLE
how to add more v rom for neogeo?

### DISCUSSION THREAD

**Discussion by odarknighto** [mamedev/discussions#169] [Q&A] [answered]
I tried to add a new sound to a kof2002 hack, but the following error occurred when reading,
Fatal error: install_ram_generic: In range 0-1ffffff mirror 0, end address is outsade of the global address mask ffffff, did you mean ffffff?
It has been added in the file neogeo.cpp, but this error appears after added. What should I do?

	ROM_REGION( 0x2000000, "cslot1:ymsnd:adpcma", 0 )
	/* Encrypted */
	ROM_LOAD( "265-v1d.bin", 0x000000, 0x800000, CRC(15e8f3f5) SHA1(7c9e6426b9fa6db0158baa17a6485ffce057d889) ) /* mask rom TC5364205 */
	ROM_LOAD( "265-v2d.bin", 0x800000, 0x800000, CRC(da41d6f9) SHA1(a43021f1e58947dcbe3c8ca5283b20b649f0409d) ) /* mask rom TC5364205 */
	ROM_LOAD( "265-v3d.bin", 0x1000000, 0x800000, CRC(255873f5) SHA1(9c9e93426b9fa7db0158baa17a6485ffce057d889))  /* mask rom TC5364205 */
	ROM_LOAD( "265-v4d.bin", 0x1800000, 0x800000, CRC(3a4156f9) SHA1(843021f1e58947dcbe3c8ca5283b20b649f0409d)  )/* mask rom TC5364205 */

**Comment by Robbbert** [comment, marked as answer]
The ym chip can only handle 0x1000000 for adpcma, and another 0x1000000 for adpcmb.

Some other emulators will let you add more, even though those sounds won't be heard. MAME doesn't permit it.

**Reply by odarknighto** [reply]
Okay, thank you for telling me. QAQ
